swap_meet/clothing.py

```diff
@@ -1,2 +1,11 @@
-class Clothing:
-    pass
+from .item import Item
+
+class Clothing(Item):
+
+    def __init__(self, category='Clothing', condition=0, age=0):
+        self.category = category
+        self.condition = condition
+        self.age = age
```

Comment:
Additionally, we're not truly using inheritance without the parent constructor. Inheritance is helpful so that child classes don't have to recreate the attributes, they can inherit attributes from their parent but still have unique values for them.